src/web/components/Onwards/OnwardsTitle.tsx

```diff
@@ -39,15 +29,16 @@ export const OnwardsTitle = ({
             {title}
         </h2>
         {description && (
-            <h3
+            <p
                 className={css`
-                    ${headline.xxxsmall()};
+                    ${headline.xxxsmall({ fontWeight: 'medium' })};
                     color: ${palette.neutral[46]};
-                    font-weight: 500;
+                    p {
+                        margin-bottom: 8px;
+                    }
                 `}
-            >
-                {description}
-            </h3>
+                dangerouslySetInnerHTML={{ __html: description }}
```

Comment:
This will need an ignore flag for the linter so we don't spam the console (even more than we currently do!)